app/src/main/java/org/mozilla/fenix/searchdialog/SearchDialogFragment.kt

```diff
@@ -97,13 +112,41 @@ class SearchDialogFragment : AppCompatDialogFragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
+        if (view.context.settings().toolbarPosition == ToolbarPosition.BOTTOM) {
+            ConstraintSet().apply {
+                clone(search_wrapper)
+
+                clear(toolbar.id, TOP)
+                connect(toolbar.id, BOTTOM, PARENT_ID, BOTTOM)
+
+                clear(awesomeBar.id, TOP)
+                clear(awesomeBar.id, BOTTOM)
+                connect(awesomeBar.id, TOP, PARENT_ID, TOP)
+                connect(awesomeBar.id, BOTTOM, toolbar.id, TOP)
+
+                applyTo(search_wrapper)
+            }
+        }
+
+        search_wrapper.setOnClickListener {
```

Comment:
okay makes sense, if you tap the blank space, dismiss the dialog

Reply:
I'm not sure if there is a better way to do this, but this is what we ended up doing on the tabs tray as well